core/src/main/java/com/datastrato/gravitino/metrics/MetricsSystem.java

```diff
@@ -33,7 +34,7 @@
 public class MetricsSystem implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(MetricsSystem.class);
   private final String name;
-  private final MetricRegistry metricRegistry;
+  @Getter private final MetricRegistry metricRegistry;
```

Comment:
I'm not sure that we should use `lombok` here. I know this is an optimization of IDEA. But maybe we should keep consistent with other places of the module `core`.

Reply:
yes, i think it is consistent with other places of the module `core` , such as `BaseMetalake` which using `lombok Getter`.

or i didn't get what you mean, can you give me more details? @qqqttt123

Reply:
1. It's ok that we don't use `lomok`.  `BaseMetalake` is related to `serialization/deserialization`. It's ok to use `lomok`. 
2. I also want to reduce unnecessary code. I want to fix this issue using the least code.

Reply:
ok, got it. So is it ok by only removing @VisibleForTesting? @qqqttt123

Reply:
> ok, got it. So is it ok by only removing `@VisibleForTesting`? @qqqttt123

hi @qqqttt123 , i have submitted a new pull request #2191 . Can you take a look in your spare time? thx.